docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Use project tests
+id: use-project-tests
+description: Custom SQL query tests
+sidebar_position: 8
+tags:
+  - how-to
+  - testing
+  - sql
+---
+
+Project tests are custom data tests that check for a passing condition, such as checking that the given query doesn’t return any unexpected rows. They are based on [dbt singular tests](https://docs.getdbt.com/docs/build/data-tests#singular-data-tests), but you don’t need to know dbt to use project tests in Prophecy. Project tests are useful for data users who know the important business questions but wouldn’t necessarily know how to write tests.
+
+Project tests perform a SQL query that checks the executed result of your project against a passing condition to see if it is successful. This is done by checking whether a generated table from your transformation meets a particular condition.
+
+Note: There could be many tests within a project. Each test is checking a table created by your choice of input Models, data sources, and transformation Gems.
+
+By default, and most often, the condition for a project test is simply a check that the given query does not return any failing rows.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/mvk1axfn7e?seo=false?videoFoam=true" title="Design a Pipeline Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
```

Comment:
THIS VIDEO IS FANTASTIC! 
One comment: why do we need the reformat? I think I missed that section of Maciej's demo. If the reformat isn't needed, I suggest we remove the reformat.

Reply:
Thank you! It took some getting used to since it's been a while since I created my last video.

It does work without the Reformat, but I think the purpose was to show a built out Model that included different gems. Creating the Model using Copilot outputs the same Model with the Reformat, so when we create a video demo using Copilot, the example will look and work the same.

Reply:
I made the edits to the video.

Reply:
Looks great! 

In the next doc I suggest renaming the Gems for clarity. I forgot to mention that earlier and it's just a small point.